src/app/shared/search/search-filters/search-filter/search-facet-filter-options/search-facet-option/search-facet-option.component.ts

```diff
@@ -119,4 +130,25 @@ export class SearchFacetOptionComponent implements OnInit {
     return getFacetValueForType(this.filterValue, this.filterConfig);
   }
 
+  /**
+   * Announces to the screen reader that the page will be reloaded, which filter has been selected
+   * and then implement “filterService.minimizeAll()”
+   */
+  selectingFilter() {
+    const filterValue = this.filterValue.value;
+    if (!this.selectedFilters.has(filterValue)) {
+      this.translateService.get('search-facet-option.update.announcement')
+        .subscribe(translatedMessage => {
```

Comment:
I *think* you can avoid doing this entire `subscribe` by following the same code structure as other components which use the LiveRegionService.  For example, see item-bistreams.service.ts: https://github.com/DSpace/dspace-angular/blob/main/src/app/item-page/edit-item-page/item-bitstreams/item-bitstreams.service.ts#L521-L525

If you follow that example, I think you can simplify all your code here to be something like this:
```
const message = this.translateService.instant('search-facet-option.update.announcement',
      { filter: filterValue });
this.liveRegionService.addMessage(message);
```